contracts/src/chainlink/solidity/emergency/StarknetValidator.sol

```diff
@@ -22,9 +22,13 @@ interface AggregatorValidatorInterface {
 /// @title StarknetValidator - makes cross chain calls to update the Sequencer Uptime Feed on L2
 contract StarknetValidator is TypeAndVersionInterface, AggregatorValidatorInterface, SimpleWriteAccessController {
   // Config for L1 -> L2 message cost approximation
+  // Message Cost = gasAdjustment * gasEstimate * gasPriceL1Feed / 100
   struct GasConfig {
+    // gas units derived from starknet estimate_message_fee
     uint256 gasEstimate;
     address gasPriceL1Feed;
+    // gasAdjustment of 100 equals 1x (see setGasConfig for more info)
+    uint32 gasAdjustment;
```

Comment:
Let's add an in-line comment about what's our default recommendation (e.g., 130).

We should also add the same check to Gauntlet commands deploying or configuring the `StarknetValidator` contract.

tldr; we should align on the default, document the default, and Gauntlet should ask the user to confirm if he's using anything other than the recommended default.

Reply:
Please also update/add info here: https://github.com/smartcontractkit/chainlink-starknet/tree/develop/docs/emergency-protocol

Reply:
Do we want to decide on 130?

Reply:
Let's settle on 130 for now, be explicit about it (docs, contract, gauntlet), and point it out to the auditors. Easy for us to change just before the launch, or update at any point.

Think about if there is a way for us to devise an integration test, which probably needs to use real mainnet contracts (L1 fast gas feed, L2 upkeep) + starknet CLI to check the current values and make sure they are above some threshold we set.